WeaponInfo.io/WeaponInfo.io/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Text;
+using System.Xml.Linq;
+
+namespace WeaponInfo.io
+{
+    class program
+    {
+
+        private static void Main()
+        {
+            Console.BackgroundColor = ConsoleColor.White;
+            Console.Clear();
+            Console.ForegroundColor = ConsoleColor.Black;
+            Console.OutputEncoding = Encoding.Unicode;
+            Console.InputEncoding = Encoding.Unicode;
+            Console.Title = "WeaponInfo.io (Докийчук А.И) ";
+            Console.WriteLine("----Weapon Info.Ua---- ");
+            StudyDataIO();
+            Console.ReadKey(true);
+
+        }
+
+        static void OutData(string TypeOfWeapon, string Name, double YearOfAdoption, string caliber, string ProdCountry)
+        {
+
+            Console.WriteLine("Даные об оружии:");
+            Console.WriteLine("\t{0,-9} {1,-9} Год производства: {2,5} Калибр: {3,10} Страна: {4,-10}", TypeOfWeapon, Name, YearOfAdoption, caliber, ProdCountry);
+            Console.ReadKey(true);
+
+
+
+
+
+
+
+
+
+
+

```

Comment:
Delete white spaces. Don't leave a lot of empty space in methods, because code is bad readable